gslib/commands/mb.py

```diff
@@ -147,8 +146,8 @@
                          for a discussion of this distinction. Default is US.
                          Locations are case insensitive.
 
-  -p proj_id             Specifies the project ID under which to create the
-                         bucket.
+  -p proj_id             Specifies the project ID or project number under which
```

Comment:
"to create the bucket under"?

Reply:
It's half a dozen of one, six of the other. Does the current wording read awkwardly for you?

Reply:
Yeah, "under which" sounds vaguely pompous. I think the last time I heard it used was in, "After completing the assignment, please place it in the bin, under which you can find your homework."

Reply:
Works for me - switched